metomi/isodatetime/data.py

```diff
@@ -309,6 +310,31 @@ def get_prev(self, timepoint: "TimePoint") -> "TimePoint":
             return prev_timepoint
         return None
 
+    def get_first_after(self, timepoint):
+        """Return the next timepoint in the series after the given timepoint
+        which is not necessarily part of the series.
+        
+        If the given timepoint is before the start point, return the
+        start point, or if it is after the end point, return None.
+        """
+        if self._get_is_in_bounds(timepoint):
+            if self.duration.is_exact():
```

Comment:
Oops, I missed one suggestion I had meant to include. `self.duration` is `None` if `self.repetitions == 1` (because that means there is only a single point in the recurrence)
```suggestion
            if self._duration is not None and self._duration.is_exact():
```

Reply:
Rolled this suggestion into another change to fix formatting